addon/components/nrg-datetime/component.js

```diff
@@ -70,8 +79,11 @@ export default Component.extend(Validation, {
     return this.timeFormat;
   }),
 
-  displayValue: computed('value', 'displayFormat', {
+  displayValue: computed('value', 'displayFormat', 'initializeDate', {
     get() {
+      if (!this.value && !this.initializeDate) {
```

Comment:
You probably can just have `!this.value` here. This is because it seems like a valid situation to have a empty value if intentional not just after the initial render.